src/duration.rs

```diff
@@ -444,9 +458,18 @@ fn div_rem_64(this: i64, other: i64) -> (i64, i64) {
 #[cfg(test)]
 mod tests {
     use super::{Duration, MIN, MAX, OutOfRangeError};
-    use std::{i32, i64};
+
+    #[cfg(feature = "std")]
```

Comment:
How'd we validate that the `no-std` implementation of, e.g. the formatting stuff, works as intended?The no-std doesn't know timezones, so needs some stuff handled separately.